build.xml

```diff
@@ -196,16 +206,11 @@
     </macrodef>
     <set-keepbrief-property test-name="${test.name}" />
 
-    <condition property="java.version.8">
-        <equals arg1="${ant.java.version}" arg2="1.8"/>
-    </condition>
-    <condition property="java.version.11">
-        <not><isset property="java.version.8"/></not>
-    </condition>
-    <fail><condition><not><or>
-        <isset property="java.version.8"/>
-        <isset property="java.version.11"/>
-    </or></not></condition></fail>
+    <fail message="Unsupported JDK version used: ${ant.java.version}">
+        <condition><not><contains string="${java.supported}" substring="${ant.java.version}"/></not></condition>
+    </fail>
+    <condition property="is.java.default"><equals arg1="${ant.java.version}" arg2="${java.default}"/></condition>
+    <echo unless:true="${is.java.default}" message="Non default JDK version used: ${ant.java.version}"/>
```

Comment:
We really don't need this warning

Reply:
This is port from OSS where it makes sense because people may build something themselves. In our case it may not be needed that much, but I think it doesn't hurt too. It may be helpful for people experimenting with the codebase to know what is really supported at the moment and where it is on them to experiment without expectations.